### PR TITLE
Updated codahale metrics dependencies to 3.2.4 #17515

### PR DESCRIPTION
**Pull request by lucianenache**
Codahale metrics libraries have been updated to 3.2.4

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

**Review by ktoso — APPROVED** [review]
Should be safe, OK from my side

**Comment by ktoso** [comment]
resolves #17515

File: project/Dependencies.scala [hunk 1]
```diff
@@ -100,8 +100,8 @@ object Dependencies {
       val jimfs        = "com.google.jimfs"            % "jimfs"                        % "1.1"              % "test" // ApacheV2
 
       // metrics, measurements, perf testing
-      val metrics         = "com.codahale.metrics"        % "metrics-core"                 % "3.0.2"            % "test" // ApacheV2
-      val metricsJvm      = "com.codahale.metrics"        % "metrics-jvm"                  % "3.0.2"            % "test" // ApacheV2
+      val metrics         = "io.dropwizard.metrics"        % "metrics-core"                 % "3.2.4"            % "test" // ApacheV2
+      val metricsJvm      = "io.dropwizard.metrics"        % "metrics-jvm"                  % "3.2.4"            % "test" // ApacheV2
```
Comment (patriknw) [inline project/Dependencies.scala]: nice, no code changes

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**